### ISSUE TITLE
sqrtint not wrapped with PARI 2.13

### ISSUE THREAD

**Issue by antonio-rojas** [sagemath/cypari2#97]
With PARI 2.13 (specifically since commit https://pari.math.u-bordeaux.fr/cgi-bin/gitweb.cgi?p=pari.git;a=commit;h=800ded9980b9af0bc0f56ed81ff76a7227369575) sqrtint is no longer being wrapped in cypari
```
In [1]: from cypari2 import Pari

In [2]: Pari().sqrtint
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-21353ce22fed> in <module>
----> 1 Pari().sqrtint

AttributeError: 'cypari2.pari_instance.Pari' object has no attribute 'sqrtint'
```

**Comment by videlec** [comment]
Because of [ 47- add optional argument to sqrtint [set remainder]  [#2229]](http://pari.math.u-bordeaux.fr/cgi-bin/gitweb.cgi?p=pari.git;a=commit;h=800ded9980b9af0bc0f56ed81ff76a7227369575) the function prototype changed to `GD&` which is currently unsupported.

**Comment by videlec** [comment]
At 3585c2c
```
>>> import cypari2
>>> pari = cypari2.Pari()
>>> pari.sqrtint(10)
3
```
Though some non-trivial adjustments are needed.

**Comment by videlec** [comment]
fixed in master (via #99)
